### PR TITLE
Replace Octicons with Codicons

### PR DESCRIPTION
**Pull request by miguelsolorio**
This replaces our use of [Octicons](https://octicons.github.com) with [Codicons](https://github.com/microsoft/vscode-codicons), which now contain all of our products icons + the new Octicons. There shouldn't be any visible changes as the icons should be the same.

Areas impacted:
- Issue reporter @RMacfarlane 
- Status bar @bpasero 
- Code lens @jrieken 
- Running extensions @alexandrudima 
- Extensions viewlet @sandy081

### CODE REVIEW

**Review by bpasero — APPROVED** [review]
@misolori I suggest you check via fulltext search where else we use "octicon":

image

Also from our build scripts in distro.

File: build/gulpfile.editor.js [hunk 1]
```diff
@@ -85,7 +85,7 @@ const extractEditorSrcTask = task.define('extract-editor-src', () => {
 			`lib.webworker.importscripts.d.ts`
 		],
 		redirects: {
-			'vs/base/browser/ui/octiconLabel/octiconLabel': 'vs/base/browser/ui/octiconLabel/octiconLabel.mock',
+			'vs/base/browser/ui/codiconLabel/codiconLabel': 'vs/base/browser/ui/codiconLabel/codiconLabel.mock',
```
Comment (jrieken) [inline build/gulpfile.editor.js]: Maybe this isn't needed anymore. I believe we have the mock for licensing reasons and I believe they are resolved now. @alexandrudima ?

**Review by jrieken — APPROVED** [review]

File: build/gulpfile.editor.js (hunk 1 above)
Comment (alexandrudima) [reply build/gulpfile.editor.js]: @jrieken Good catch. This is a bad change, the standalone editor needs to ship codicons

@misolori please just delete this line.

**Review by alexdima — CHANGES_REQUESTED** [review]
Just a small change in gulpfile.editor to have the standalone editor ship icons...

File: build/gulpfile.editor.js (hunk 1 above)
Comment (misolori) [reply build/gulpfile.editor.js]: Done

**Pull request merged**